public/index.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+/**
+ * Laravel - A PHP Framework For Web Artisans
+ *
+ * @package  Laravel
+ * @author   Taylor Otwell <taylor@laravel.com>
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Tag value for @author tag indented incorrectly; expected 2 spaces but found 3](https://app.codacy.com/gh/BuildForSDG/Team-059-Product/pullRequest?prid=5478046)